### ISSUE TITLE
Scene is missing

### ISSUE THREAD

**Issue by martdob** [leovd100/Music-Project-C-.NET#1]
Hi, are you still working on this project?
The provides scene seems to be incomplete. Also, the code isn't compatible with Unity 2019+.

**Comment by leovd100** [comment]
yes, i am working on the project, but very little. I am working internally.
Sorry, but some files I didn't upload.
It uses the Unity 5.2 ~ 5.6. Not compatible with Unity 2019+
I used this unity because I couldn't use 2017+.
